ras_backstage/views/backstage_view.py

```diff
@@ -23,13 +27,14 @@ def get_pw(username):
 
 @backstage_view.route('/sign_in', methods=['POST'])
 def sign_in():
+    logger.info("Sign-in POST request.")
```

Comment:
Is it worth logging this at info?

Reply:
Tbf that log isn't really needed because http requests to endpoints are logged anyway, was just following how another service did it might get rid of it completely.